shared/src/main/scala/SugarMap.scala

```diff
@@ -0,0 +1,47 @@
+package vyxal
+object SugarMap:
+  def internalMap: Map[String, String] =
```

Comment:
Should this be a val?

Reply:
I wasn't sure if it should be a def or a val. If it can be a val and not impact anything, then sure